views/pages/players.html

```diff
@@ -12,4 +12,95 @@
   </tr>
   {{end}}
 </table>
+<div class="flex min-h-full flex-col justify-center px-6 py-12 lg:px-8">
+  <div class="sm:mx-auto sm:w-full sm:max-w-sm">
+    <img class="mx-auto h-10 w-auto" src="/public/icons/soccer-football-svgrepo-com.svg" alt="Scouting Pro">
+    <h2 class="mt-10 text-center text-2xl font-bold leading-9 tracking-tight text-gray-900">Add a player</h2>
+  </div>
+
+  <div class="mt-10 sm:mx-auto sm:w-full sm:max-w-sm">
+    <form class="space-y-6" hx-post="/players" hx-target="#result" hx-swap="innerHTML">
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="name" class="block text-sm font-medium leading-6 text-gray-900">Full Name:</label>
+          <div class="mt-2">
+            <input type="text" id="name" name="name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="birthdate" class="block text-sm font-medium leading-6 text-gray-900">Birthdate (yyyy-mm-dd):</label>
+          <div class="text-sm">
+            <input type="date" id="birthdate" name="birthdate" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="height" class="block text-sm font-medium leading-6 text-gray-900">Height (cm):</label>
+          <div class="text-sm">
+            <input type="number" id="height" name="height" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+          </div>
+        </div>
+      </div>
+      <div>
+        <div class="flex items-center justify-between">
+          <label for="weight" class="block text-sm font-medium leading-6 text-gray-900">Weight (grams):</label>
```

Comment:
Like with the birthdate above, perhaps kilos would be a better human readable input? We can convert to grams internally for the database, or just change the comment on the `weight` field in `database.Player` to say kgs.

Reply:
Are lbs or kgs more commonly used? I think we should stick with whichever is most widely used and not have to worry about conversions or rounding.

Reply:
switched to lbs